cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,35 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
```

Comment:
Ah, right. We've been specifying types for functions but not (as a rule) for the code within them. This is kind of sort of following the [PREFER providing type annotations on public APIs](https://www.dartlang.org/articles/style-guide/#type-annotations) guideline, except for the fact that we're not really defining a public API here. I suppose it makes sense to drop the Future if you're not explicitly returning anything. (On the other hand, it does let people know that there really is a value returned. I'm not sure what to do. @munificent?)